files/default/squid.conf

```diff
@@ -0,0 +1,23 @@
+acl to_localnet dst 0.0.0.1-0.255.255.255 # RFC 1122 "this" network (LAN)
+acl to_localnet dst 10.0.0.0/8            # RFC 1918 local private network (LAN)
+acl to_localnet dst 100.64.0.0/10         # RFC 6598 shared address space (CGN)
+acl to_localnet dst 169.254.0.0/16        # RFC 3927 link-local (directly plugged) machines
+acl to_localnet dst 172.16.0.0/12         # RFC 1918 local private network (LAN)
+acl to_localnet dst 192.168.0.0/16        # RFC 1918 local private network (LAN)
+acl to_localnet dst fc00::/7              # RFC 4193 local private network range
+acl to_localnet dst fe80::/10             # RFC 4291 link-local (directly plugged) machines
+
+acl SSL_ports port 443
+acl Safe_ports port 80
+acl Safe_ports port 443
```

Comment:
can you add 8080, 8443 to the list of safe ports and safe ssl ports.

Reply:
When would we need those?

Reply:
someones website might not be on the normal ports.